lib/new_version.dart

```diff
@@ -252,12 +258,16 @@ class NewVersion {
   }
 
   /// Launches the Apple App Store or Google Play Store page for the app.
-  void _launchAppStore(String appStoreLink) async {
+  Future<void> _launchAppStore(String appStoreLink) async {
     debugPrint(appStoreLink);
```

Comment:
Yeah, I'm thinking we just make this function public, and make it take a `VersionStatus` parameter.